runtime/compiler/env/J2IThunk.cpp

```diff
@@ -107,6 +107,7 @@ char TR_J2IThunkTable::terseTypeChar(char *type)
       {
       case '[':
       case 'L':
+      case 'Q': //TODO: further consideration may be needed
```

Comment:
I know I suggested adding a "TODO", but I understand that some people prefer that issues be opened to track outstanding problems instead.  Please remove the "TODO", and we'll open an issue in the enterprise Github to track the additional work that might be needed.